demo/docker/docker-compose.yml

```diff
@@ -1,28 +1,24 @@
 version: '2.1'
 
 services:
-
   app:
-    build:
-      context: ./app
-      args:
-        #TODO: consider sth like:
-        #curl -s https://api.github.com/repos/touk/nussknacker/releases/latest | grep tag_name | awk '{ print $2 }' | sed -e 's/"//g' -e 's/,//g'
-        #to retrieve latest version
-        version: ${NUSSKNACKER_VERSION}
+    image: ${NUSSKNACKER_IMAGE}:${NUSSKNACKER_VERSION}
+    entrypoint: ["/opt/nussknacker/bin/wait-for-it.sh"]
+    command: ["-h", "taskmanager", "-p", "6126", "-t", "60", "--", "/opt/nussknacker/bin/docker-run.sh"]
```

Comment:
let's fix NK if it fails on waiting for taskmanager, but we should rely on tm being available...
besides - we should be waiting for jobmanager, not task manager?

Reply:
When we set `queryableStateProxyUrl: "taskmanager:9069"` and it's not avaiable then application throws exception and doesn't work.